docs/admin/integrations/infoblox_setup.md

```diff
@@ -128,6 +128,23 @@ The above filters will allow the loading of the following data from Infoblox and
 - Only IPv4 prefixes and IP addresses, contained within the `192.168.0.0/16` container, located in Infoblox network view "dev" and Nautobot namespace "dev". All IPv6 prefixes and IP addresses in the Infoblox network view "dev" and Nautobot namespace "dev".
 - Only IPv4 prefixes and IP addresses, contained within the `10.0.0.0/8` container, located in Infoblox network view "test" and Nautobot namespace "test".  Only IPv6 prefixes and IP addresses contained withing the `2001:5b0:4100::/40` container that are located in the Infoblox network view "test" and Nautobot namespace "test".
 
+### Configuring Infoblox Network Views to Nautbot Namespace map
+
+This setting allows you to create mappings between Infoblox network views and Nautobot namespaces. By default, the **default** network view maps to the **Global** Nautobot namespace.
+
+The purpose of this setting is to provide flexibility, enabling you to:
+- Assign a different namespace instead of **Global** for the default Infoblox network view.
```

Comment:
for mkdocs you need a space before bullets to render properly.

Reply:
Weird that markdownlint says the opposite: https://github.com/DavidAnson/markdownlint/blob/v0.37.4/doc/md007.md

Reply:
This would be the better reference: https://pymarkdown.readthedocs.io/en/latest/plugins/rule_md007/#python-markdown-support

Reply:
Re-reading what the pymarkdown docs say it appears to be that you just need spacing above the list, not before the dash/asterisk. I've pushed a fix to add that additional spacing around the list so this should be resolved.